extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/PanacheQuery.java

```diff
@@ -40,6 +41,20 @@
      */
     public <T> PanacheQuery<T> project(Class<T> type);
 
+    /**
+     * Generates a SELECT clause with the specified properties.
+     * <p>
+     * The row mapper function is used to build a DTO object for each row of the result. A result row
+     * is represented by {@code Object[]}.
+     * 
+     * @param <T>
+     * @param rowMapper Maps a result row to a DTO object.
+     * @param properties
+     * @return a new query with the same state as the previous one (params, page, range, lockMode, hints, ...).
+     * @see #project(Class)
```

Comment:
Can you also add a `@see` on the doc of the other projet method for a better discoverability ?